tests/python_package_test/test_arrow.py

```diff
@@ -183,3 +186,32 @@ def test_dataset_construct_weights(array_type: Any, weight_data: Any, arrow_type
 
     expected = np.array([3, 0.7, 1.5, 0.5, 0.1], dtype=np.float32)
     np_assert_array_equal(expected, dataset.get_weight(), strict=True)
+
+
+# -------------------------------------------- GROUPS ------------------------------------------- #
+
+
+@pytest.mark.parametrize(
+    ["array_type", "group_data"], [(pa.array, [2, 3]), (pa.chunked_array, [[2], [3]])]
+)
+@pytest.mark.parametrize(
+    "arrow_type",
+    [
+        pa.int8(),
+        pa.int16(),
+        pa.int32(),
+        pa.int64(),
+        pa.uint8(),
+        pa.uint16(),
+        pa.uint32(),
+        pa.uint64(),
+    ],
+)
+def test_dataset_construct_groups(array_type: Any, group_data: Any, arrow_type: Any):
```

Comment:
```suggestion
def test_dataset_construct_groups(array_type, group_data, arrow_type):
```

Please don't add type hints on `pytest` test functions in this project. Especially when they're all set to `Any`, they just make the code a little harder to read without providing any documentation or help to type-checkers.

Can you please remove them? And please also remove them from the test cases introduced in the previous Arrow PRs? Sorry I missed that before.